proto/mrt/mrt.c

```diff
@@ -456,8 +456,31 @@ mrt_rib_table_entry(struct mrt_table_dump_state *s, rte *r)
       mrt_log(s, "Attribute list too long for %N", r->net->n.addr);
       alen = 0;
     }
+    void * len_pos = b->pos - 2;        /* save position of length byte */
```

Comment:
this is the actual code change to write a BGP_REACH_NLRI with a nexthop to the mrt dump